tdvt/tdvt/exprtests/standard/setup.cast.str.datetime.txt

```diff
@@ -0,0 +1,9 @@
+DATETIME('1900-01-01 01:00:00')
+DATETIME('1900-12-30 12:00:00')
+DATETIME('1900-12-31 23:59:59')
+DATETIME('2018-01-01 01:00:00')
+DATETIME('2018-12-29 12:00:00')
+DATETIME('2018-12-31 23:59:59')
+
+DATETIME(STR([datetime0]))
+DATETIME(STR([datetime1]))
```

Comment:
This test case is failing on a number of ds that cannot handle the cast on NULL or empty string. Let's split it off into a new test file maybe? (`setup.cast.str.datetime.empty`)